modules/ding_news/ding_news.pages_default.inc

```diff
@@ -314,145 +635,177 @@ taxonomy/term/%term:tid',
-    $pane->pid = 'new-64a515e4-fb15-435d-bcf6-241785791333';
+    $pane->pid = 'new-af0b4dab-20c3-42c8-b489-aebf3fda7a8c';
     $pane->panel = 'left_sidebar';
-    $pane->type = 'taxonomy_menu';
-    $pane->subtype = 'taxonomy_menu';
+    $pane->type = 'page_title';
+    $pane->subtype = 'page_title';
     $pane->shown = TRUE;
     $pane->access = array();
     $pane->configuration = array(
-      'slug' => 'nyheder',
-      'default_link_text' => 'All categories',
-      'context' => array(
-        0 => 'argument_term_1',
-        1 => 'empty',
-      ),
-      'override_title' => 0,
-      'override_title_text' => '',
-      'dropdown' => 0,
-      'vocabulary' => 'news_category',
-      'base' => '',
+      'markup' => 'h2',
+      'class' => '',
+      'id' => '',
+      'override_title' => 1,
+      'override_title_text' => 'Nyheder',
+      'override_title_heading' => 'h2',
     );
     $pane->cache = array();
     $pane->style = array(
       'settings' => NULL,
     );
-    $pane->css = array(
-      'css_id' => '',
-      'css_class' => '',
-    );
+    $pane->css = array();
     $pane->extras = array();
     $pane->position = 0;
     $pane->locks = array();
-    $pane->uuid = '64a515e4-fb15-435d-bcf6-241785791333';
-    $display->content['new-64a515e4-fb15-435d-bcf6-241785791333'] = $pane;
-    $display->panels['left_sidebar'][0] = 'new-64a515e4-fb15-435d-bcf6-241785791333';
+    $pane->uuid = 'af0b4dab-20c3-42c8-b489-aebf3fda7a8c';
+    $display->content['new-af0b4dab-20c3-42c8-b489-aebf3fda7a8c'] = $pane;
+    $display->panels['left_sidebar'][0] = 'new-af0b4dab-20c3-42c8-b489-aebf3fda7a8c';
     $pane = new stdClass();
-    $pane->pid = 'new-f761af87-a40a-4bcf-99c2-0320ab254d4e';
+    $pane->pid = 'new-1b5092b3-c40d-4e8a-8608-0828852e43dd';
     $pane->panel = 'left_sidebar';
-    $pane->type = 'campaign';
-    $pane->subtype = 'campaign';
+    $pane->type = 'menu_title';
+    $pane->subtype = 'menu_title';
+    $pane->shown = FALSE;
+    $pane->access = array();
+    $pane->configuration = array(
+      'override_title' => 1,
+      'override_title_text' => 'Nyheder',
+      'override_title_heading' => 'h2',
+    );
+    $pane->cache = array();
+    $pane->style = array(
+      'settings' => NULL,
+    );
+    $pane->css = array();
+    $pane->extras = array();
+    $pane->position = 1;
+    $pane->locks = array();
+    $pane->uuid = '1b5092b3-c40d-4e8a-8608-0828852e43dd';
+    $display->content['new-1b5092b3-c40d-4e8a-8608-0828852e43dd'] = $pane;
+    $display->panels['left_sidebar'][1] = 'new-1b5092b3-c40d-4e8a-8608-0828852e43dd';
+    $pane = new stdClass();
+    $pane->pid = 'new-64a515e4-fb15-435d-bcf6-241785791333';
+    $pane->panel = 'left_sidebar';
+    $pane->type = 'taxonomy_menu';
+    $pane->subtype = 'taxonomy_menu';
     $pane->shown = TRUE;
     $pane->access = array();
     $pane->configuration = array(
-      'ding_campaign_count' => 3,
-      'ding_campaign_offset' => 0,
+      'slug' => 'nyheder',
```

Comment:
The slugs need to match the path of the page the menu links to.
It seems to be consistent with previous versions of the file.